### PR TITLE
Enhancements to Java config

### PR DESCRIPTION
**Pull request by liebowitz**
- Copy default maven global settings

### CODE REVIEW

File: files/maven_settings.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd" xmlns="http://maven.apache.org/SETTINGS/1.1.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <mirrors>
+    <mirror>
+      <mirrorOf>external:*</mirrorOf>
+      <name>remote-repos</name>
+      <url>https://artifactory.automate.datapipe.net:443/remote-repos</url>
```
Comment (chrislovecnm) [inline files/maven_settings.xml:8]: Any way to not put in backend url?

**Review by chrislovecnm — CHANGES_REQUESTED** [review]
Just a question about URLs

File: files/maven_settings.xml (hunk 1 above)
Comment (liebowitz) [reply files/maven_settings.xml:8]: I'd prefer to externalize it as well, but I don't believe it is possible in this case...

Comment (chrislovecnm) [reply files/maven_settings.xml:8]: ENV vars?

**Comment by liebowitz** [comment]
Closing and moving config to platform parent POM.

**Pull request closed without merging**